packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -868,6 +868,72 @@ const defaultConfigs = {
       ETHBTC_PERP: { type: "uniswap", invertPrice: true, uniswapAddress: "0x899a45ee5a03d8cc57447157a17ce4ea4745b199" },
       PERP_FRM: { type: "frm", perpetualAddress: "0x32f0405834c4b50be53199628c45603cea3a28aa" }
     }
+  },
+  BALUSD: {
+    type: "expression",
+    expression: `
+      BALANCER_ETH_INVERTED = 1 / SPOT_BALANCER_ETH;
+      SPOT_BALANCER = BALANCER_ETH_INVERTED * ETHUSD;
```

Comment:
Curious why you don't just set `invertPrice:true` on the SPOT_BAlANCER_ETH customFeed?

Reply:
So `invertPrice` actually shouldn't actually be in the SPOT_BALANCER_ETH customFeed, since the Balancer price feed does not accept an `invertPrice` param from what I can see. What I should've done is just switch the `balancerTokenIn` and `balancerTokenOut` addresses. I'll update to do that and remove the invert in the expression.

Reply:
Tested and updated without inverting in the expression.